app/components/user-search.cjsx

```diff
@@ -14,7 +14,7 @@ module.exports = React.createClass
       apiClient.type('users').get display_name: "#{value}*", page_size: 10
         .then (users) =>
           opts = for user in users
-            { value: user.id, label: "@#{ user.login }: #{ user.display_name }" }
+            { value: user.id, label: "@#{ user.login }" }
```

Comment:
Ah ok. I removed it because it looked a bit silly as everyone's login and display_name were identical.

Given the other methods of referencing users use login (tagging, mentions), shouldn't the search do so as well?